https://bugs.gentoo.org/903657
https://github.com/tmate-io/tmate/pull/281

From a5c6e80d3c54cd7faed52de5283b4f96bea86c13 Mon Sep 17 00:00:00 2001
From: Carlo Cabrera <30379873+carlocab@users.noreply.github.com>
Date: Sun, 5 Mar 2023 20:58:13 +0800
Subject: [PATCH] Fix finding msgpack 6+

`msgpack.pc` was renamed to `msgpack-c.pc` upstream in msgpack/msgpack-c#1053.
--- a/configure.ac
+++ b/configure.ac
@@ -209,6 +209,17 @@ PKG_CHECK_MODULES(
   ],
   found_msgpack=no
 )
+# msgpack.pc was renamed to msgpack-c.pc in 6.0.0.
+PKG_CHECK_MODULES(
+  MSGPACKC,
+  msgpack-c >= 1.1.0,
+  [
+    CPPFLAGS="$MSGPACKC_CFLAGS $CPPFLAGS"
+    LIBS="$MSGPACKC_LIBS $LIBS"
+    found_msgpack=yes
+  ],
+  found_msgpack=no
+)
 if test "x$found_msgpack" = xno; then
   AC_MSG_ERROR("msgpack >= 1.1.0 not found")
 fi

